src/test/config.sol

```diff
@@ -99,9 +99,9 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0xD8E6fBb43354dd6637561208eF5Db1a62e6408DC),        // populate with deployed spell if deployed
-            deployed_spell_created:         1651237244,        // use get-created-timestamp.sh if deployed
-            previous_spell:                 address(0),        // supply if there is a need to test prior to its cast() function being called on-chain.
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
+            deployed_spell_created:         0,                 // use get-created-timestamp.sh if deployed
+            previous_spell:                 address(0xD8E6fBb43354dd6637561208eF5Db1a62e6408DC),        // supply if there is a need to test prior to its cast() function being called on-chain.
```

Comment:
This shouldn't be necessary as prev spell was already cast.

Reply:
Done.